APCAppCore/APCAppCore/UI/TabBarControllers/Activities/APCActivitiesViewController.m

```diff
@@ -107,12 +112,19 @@ - (void) viewDidLoad
 
     // make sure we know the state of CoreMotion permissions so it's available when we need it
     [self.permissionManager requestForPermissionForType:kAPCSignUpPermissionsTypeCoremotion withCompletion:nil];
+
+    NSUserDefaults *defaults = [NSUserDefaults standardUserDefaults];
```

Comment:
Please remove this. This is shared code between different apps. A default should be NO with the option for a given app to override and set to YES.